presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -1110,6 +1110,11 @@ protected Scope visitUnnest(Unnest node, Optional<Scope> scope)
             ImmutableList.Builder<Field> outputFields = ImmutableList.builder();
             for (Expression expression : node.getExpressions()) {
                 ExpressionAnalysis expressionAnalysis = analyzeExpression(expression, createScope(scope));
+                if (!expressionAnalysis.getScalarSubqueries().isEmpty()) {
+                    throw new SemanticException(NOT_SUPPORTED,
+                            node,
+                            "Scalar subqueries in UNNEST are not yet supported");
```

Comment:
I think we can get rid of word "yet"
Scalar subqueries in UNNEST are not supported